data/CMakeLists.txt

```diff
@@ -56,6 +56,9 @@ if(NOT WIN32)
   if("webp" IN_LIST DT_SUPPORTED_EXTENSIONS)
     list(APPEND DESKTOP_MIME_TYPES image/webp)
   endif()
+  if("qoi" IN_LIST DT_SUPPORTED_EXTENSIONS)
+    list(APPEND DESKTOP_MIME_TYPES image/qoi)
+  endif()
```

Comment:
Don't think we should be adding this mime type just yet - it is not yet registered, and also not present in https://gitlab.freedesktop.org/xdg/shared-mime-info/-/blob/master/data/freedesktop.org.xml.in

Reply:
I strongly disagree. The author himself insists on this mime type. See https://github.com/phoboslab/qoi#mime-type-file-extension

Quoting here:
"The recommended MIME type for QOI images is image/qoi. While QOI is not yet officially registered with IANA, I believe QOI has found enough adoption to prevent any future image format from choosing the same name, thus making a MIME type collision highly unlikely (https://github.com/phoboslab/qoi/issues/167)."

Also please read https://github.com/phoboslab/qoi/issues/167#issuecomment-1117240154